extensions/amp-analytics/0.1/vendors.js

```diff
@@ -2048,6 +2048,30 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'scroll': {
+    'enabled': 'AUTHDATA(scroll.scroll)',
+    'requests': {
+      'base': 'https://connect.scroll.com/amp/analytics',
+      'scroll': '${base}?rid=ACCESS_READER_ID'
+                + '&cid=CLIENT_ID(cid-fallback-cookie)'
+                + '&o=SOURCE_URL'
+                + '&c=CANONICAL_URL'
+                + '&d=AUTHDATA(scroll.scroll)'
+                + '&v=AUTHDATA(scroll.visitId)'
+                + '&s=${totalEngagedTime}',
+    },
+    'triggers': {
+      'trackInterval': {
+        'on': 'timer',
+        'timerSpec': {
+          'interval': 15,
+          'maxTimerLength': 7200,
```

Comment:
According to the other PR you have. You only want to send analytics data after user log in.
But `timer` trigger starts right after `<amp-analytics>` is ready. Want to make sure what's your desired behaviors with timer trigger.